cupyx/scipy/fftpack/fft.py

```diff
@@ -112,20 +113,31 @@ def get_fft_plan(a, shape=None, axes=None, value_type='C2C'):
         raise ValueError('C2R/R2C PlanNd for F-order arrays is not supported')
 
     # generate plan
+    # (load from cache if it exists, otherwise create one but not cache it)
     if n > 1:  # ND transform
         out_size = _get_fftn_out_size(
             shape, transformed_shape, axes[-1], value_type)
+        # _get_cufft_plan_nd handles the interaction with plan cache
         plan = _get_cufft_plan_nd(
-            shape, fft_type, axes=axes, order=order, out_size=out_size)
+            shape, fft_type, axes=axes, order=order, out_size=out_size,
+            to_cache=False)
```

Comment:
Is the assumption here that the reason to call `get_fft_plan` is when you want to handle the plans yourself and so it should not be cached? I think that sounds reasonable.

Reply:
That's right. Originally I made it cached here, but I changed my mind as the intention of `get_fft_plan()` was to allow users to manage plans themselves.